src/mongocxx/options/ssl.hpp

```diff
@@ -39,9 +39,12 @@ class MONGOCXX_API ssl {
     void ca_file(std::string ca_file);
     const bsoncxx::stdx::optional<std::string>& ca_file() const;
 
-    void ca_dir(std::string ca_file);
+    void ca_dir(std::string ca_dir);
```

Comment:
Added! I discovered in reading our docs that we had named an option the way the C driver did instead of the way the docs do. I updated it to reflect the new docs.